resources/features/Sampling.feature

```diff
@@ -0,0 +1,106 @@
+Feature: Making a Sample
+    The sample creation method will ingest a frame and a set of stratification parameters to produce an output sample.
+    To do this, the pre-filtered dataset is sorted in ascending order by 'prn'. Each of the stratification stratas with
+    a selection type parameter set as 'Census' or 'Prn Sampling' (ommitting 'Universal') is then processed. We then use
+    the start point parameter (of a given strata), and select units in prn order until the required sample size is
+    reached. All samples are outputted collectively as a DataFrame, with an appended 'Cell Number' to each to indicate
+    the strata to which it belongs.
+
+    Scenario Outline: A strata for Census is created
+        Given a frame and a stratification properties file paths:
+            | data_input_path   | strat_properties        | expected_output   |
+            | <input_path>      | <strat_properties_path> | <output_path>     |
+        When a <language> Sample is created from the pre-filtered frame
+        Then a Sample DataFrame containing the Census strata is returned and exported to CSV
```

Comment:
How can the business use this test to confirm that stratification has been applied correctly?

I appreciate that a certain style has been adopted by the existing Cucumber tests - but generally assertions that are hidden behind general steps such as "then a sample dataFrame is returned" are not the spirit of Cucumber.  The business has no way of confirming the step.

The preferred approach is for the Then step to define a table that contains the expected output.  The business then has input into the content of that table - and has all the information it needs in the test itself to confirm that the code has done exactly what they wanted it to.